examples/vercel-ai-chatbot/package.json

```diff
@@ -24,8 +24,10 @@
     "@vercel/analytics": "^1.1.2",
     "@vercel/kv": "^1.0.1",
     "@vercel/og": "^0.6.2",
+    "cdktf-cli": "^0.20.6",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.0",
+    "constructs": "^10.3.0",
```

Comment:
this shouldn't be needed, but cdktf-cli and cdktf have different versions of `constructs` in ways that aren't compatible -.-